src/commands/dev.ts

```diff
@@ -363,10 +364,12 @@ export async function command(commandOptions: CommandOptions) {
         config,
       });
       builtFileResult.result = await transformFileImports(
-        builtFileResult.result,
-        // This is lame: because routes don't have a file extension, we create
-        // a fake file name with the correct extension, which is the important bit.
-        'file.' + (srcFileExtensionMapping[ext] || ext),
+        {
+          locOnDisk: fileLoc,
+          code: builtFileResult.result,
+          baseExt: srcFileExtensionMapping[baseExt] || baseExt,
+          expandedExt,
+        },
```

Comment:
We don’t need this logic anymore! 🎉 Now that we are relying on `baseExt`, we can use that as the primary lifter of “how do we handle this file” rather than relying on tenuous logic based on file name.

Reply:
Thanks for cleaning up my mess!